dev_env_setup.sh

```diff
@@ -11,38 +11,32 @@
 pip install --upgrade pip
 pip install -r bin/requirements.txt
 
-pip install -e python_modules/dagster
-pip install -r python_modules/dagster/requirements.txt
+pip install -e python_modules/dagster[aws]
```

Comment:
?

dagster_aws

Reply:
Sadly not. We still have https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/setup.py#L83, without which the S3-backed object store etc. won't work